ui-ngx/src/app/modules/home/components/profile/tenant-profile.component.ts

```diff
@@ -59,7 +59,7 @@ export class TenantProfileComponent extends EntityComponent<TenantProfile> {
         consumerPerPartition: true,
         name: 'Main',
         packProcessingTimeout: 2000,
-        partitions: 10,
+        partitions: 2,
```

Comment:
I would like to control a **retention policy** for isolated topic.
With 3k topics with a default limit 1Gb and 7 days, we will end up with **3 TeraByte** * replication factor PVC size !!!!
Retention policy like this
![image](https://github.com/thingsboard/thingsboard/assets/79898499/52c3aca0-e5b1-4920-821d-c8a6689049dc)

That is a lot!